app/app.iml

```diff
@@ -1,110 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Every attempt I've made to remove this file breaks the project in Android Studio. We need to keep it.